babel.config.js

```diff
@@ -17,6 +20,9 @@ module.exports = function generateConfig(api) {
     ],
     ignore: ['__mocks__'],
     plugins: [
+      ...(BABEL_ENV === 'production'
```

Comment:
Will this file ever be read _not_ in production? Storybook ships with it's own babel config, and there's an outstanding bug that `babel.config.js` files aren't sourced - https://github.com/storybookjs/storybook/issues/6633#issuecomment-553397732

I wonder if we could just move the production conditionals to be the default? We can additionally change the script in `package.json` to not set the `BABEL_ENV` to `production`

Reply:
When the bug in storybook is fixed it will use our bable config in which case we will need to specify then environment

Reply:
Good point, plus as you mentioned to me it's being used for jest, which is a non production env.